spec/index.adoc

```diff
@@ -3910,17 +3948,17 @@ Read response::
 * If `F(Q.Arg, Q.sender, Env) = Trap trap` then
 +
 ....
-{status: rejected; reject_code: CANISTER_ERROR, reject_message: <implementation-specific>, error_code: <implementation-specific>}
+{status: rejected; reject_code: CANISTER_ERROR, reject_message: <implementation-specific>, error_code: <implementation-specific>, node_signatures: <implementation-specific>}
```

Comment:
Not really implementation specific, is it? It's specified above what it should be. Maybe just write `[]` to omit from the example.

Reply:
> Not really implementation specific, is it?

The exact value is implementation-specific because it depends on how the BN chooses the replica(s) for executing the query.

> Maybe just write [] to omit from the example.

This would constraint an implementation to never include any signature into the response.

Reply:
Ok, fine with me